RedditSharp/Things/Post.cs

```diff
@@ -108,7 +108,7 @@ public Comment[] Comments
         public Uri Permalink { get; set; }
 
         [JsonProperty("score")]
-        public int Score { get; set; }
+        public new int Score { get; set; }
```

Comment:
This whole property should likely just be removed as it's already inherited from VoteableThing and that's what the warning was about. It doesn't need to be overridden / hidden